### ISSUE TITLE
Mobilenetv2 - model card

### ISSUE THREAD

**Issue by keerthana-r-mcw** [tenstorrent/tt-metal#13798]
**Brief Exec summary of mobilenetv2**
- Bring up PR: https://github.com/tenstorrent/tt-metal/pull/18953
- Perf PR : https://github.com/tenstorrent/tt-metal/pull/20396
- Demo PR: https://github.com/tenstorrent/tt-metal/pull/20498
- Supported Resolutions: 224, 256, 512
- Supported Batch_size: 9 for 224 res, 7 for 256, 1 for 512
- Ref [link](https://github.com/pytorch/vision/blob/main/torchvision/models/mobilenetv2.p)
- Unit_test : [sabira/mobilenetv2_conv_unit_test_256](https://github.com/tenstorrent/tt-metal/tree/sabira/mobilenetv2_conv_unit_test_256)

- PCC:  0.937(224 res)
- Device FPS - 699
- e2e perf with trace - 2470 FPS (bs=10)
- Demo results :[Imagenet_dataset_result]( https://docs.google.com/spreadsheets/d/1PEjzIucgefp7pstCpB5SENp3F1EJhUh8o-UYbY9GhTY/edit?gid=338649004#gid=338649004)
- Imagenet_accuracy - 0.68 (bs = 8, res =224)
- Issues :
          1. pcc - [#18409](https://github.com/tenstorrent/tt-metal/issues/18409)
          2. trace -  [#18585](https://github.com/tenstorrent/tt-metal/issues/18585) (CLOSED)

Graph - [mobilenetv2.pdf](https://github.com/user-attachments/files/17373535/mobilenetv2.pdf)

Submodules:
- Conv2dNormActivation
- InvertedResidual

Ops:
- Conv2d
- Batchnorm2d
- ReLu6
- Linear
- adaptive_avg_pool2d
- Flatten

**Comment by mbahnasTT** [comment]
<img width="1556" alt="Image" src="https://github.com/user-attachments/assets/cd6bbac1-52b9-42ce-beb7-8c5377b5ad1e" />
